src/controller/java/src/chip/jsontlv/TlvToJson.kt

```diff
@@ -55,6 +55,38 @@ fun TlvReader.toJsonString(): String {
   }
   return getStructJson().toString()
 }
+
+/**
+ * Encodes TLV into kotlin Object. If the TLV reader is positioned TLV Structure, Object will return
+ * to json format.
+ */
+fun TlvReader.toAny(tag: Tag = AnonymousTag): Any? {
```

Comment:
toAny means to convert tlv to  Kotlin primitive type + array type, [TlvToJson.kt](https://github.com/project-chip/connectedhomeip/pull/28767/files#diff-c498ce623aa3cd7e7f3fe3660d241de3c25b7da3400e2596a8889f5a74c5aca9) is to convert tlv to json only
it is not really suitable to put this functionality into tlvtojson lib

Reply:
or you can relocate it to tlv folder instead of jsontlv. thanks

Reply:
thanks